org.d3cod3.Mosaic.yml

```
@@ -0,0 +1,431 @@
+app-id: org.d3cod3.Mosaic
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: mosaic
+# rename-icon: accessories-dictionary
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=host
+# cleanup:
+#  - /include
+add-extensions:
+    org.freedesktop.Platform.ffmpeg-full:
+      directory: lib/ffmpeg
+      version: 20.08
```

```suggestion
      version: '20.08'
```
Remember, this is yaml & the versions are strings, not dates.